kolibri/deployment/default/settings/base.py

```diff
@@ -62,6 +62,7 @@
     'kolibri.core.auth.apps.KolibriAuthConfig',
     'kolibri.core.content',
     'kolibri.core.logger',
+    'kolibri.core.sqlite',
```

Comment:
Let's avoid another application. Can this live in `kolibri.core.deviceadmin` ?

Reply:
I wrote  it in sqlite after asking for Richard's advice. Deviceadmin also can make sense, could you discuss it with @indirectlylit & @rtibbles  to avoid having to move it backwards and forwards again?

Reply:
If we don't manage to divide our various management commands into meaningful application spaces, we'll end up with way too many applications, and we'll have a pattern where we invent an application for simple tasks instead of encouraging reuse etc. between various utility functions. A Django application is best defined by also having models etc -- but since we did have a couple of things that didn't require models, we have bundled them into `deviceadmin`. IMHO of course :)

Reply:
Doesn't matter one way or the other to me. (IMO we should not be exposing Django `manage` management commands directly to end users anyway, but that's a completely different conversation for #3727)

Putting it in `deviceadmin` seems reasonable as a way to prevent app bloat if that's a concern

Reply:
> we should not be exposing Django manage management commands directly to end users anyway

I'm :+1: with this - just that I find it fine to keep such functions wrapped as convenient management commands for advanced users -- then later on, we can build UIs that invoke them.

Reply:
Perfect, movement done.